src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresDatatypeIT.java

```diff
@@ -133,14 +183,20 @@ private void createTableWithUuidColumns() throws SQLException {
     log.info("Created table {} with UUID column", tableName);
   }
 
-  private void startTask() {
-    task = new JdbcSinkTask();
-    task.start(props);
-  }
-
-  public void stopTask() {
-    if (task != null) {
-      task.stop();
+  private void createTableWithLessFields() throws SQLException {
+    log.info("Creating table {} with UUID column", tableName);
+    try (Connection c = pg.getEmbeddedPostgres().getPostgresDatabase().getConnection()) {
+      c.setAutoCommit(false);
```

Comment:
Why the need for this and explicit commit right after execute?

Reply:
Which part are you referring to? This method is basically the same as the above one, but with fewer fields to simulate a schema exception to test the error reporting functionality.

Reply:
Oh, got it.